buildSrc/src/main/java/com/toggl/buildsrc/dependencies.kt

```diff
@@ -67,6 +67,12 @@ object Libs {
         const val browser = "androidx.browser:browser:1.0.0"
         const val fragment = "androidx.fragment:fragment:1.0.0"
 
+        object Hilt {
+            private const val version = "1.0.0-alpha01"
+            const val viewModel = "androidx.hilt:hilt-lifecycle-viewmodel:$version"
+            const val compiler = "com.google.dagger:hilt-android-compiler:$version"
```

Comment:
```suggestion
            const val compiler = "androidx.hilt:hilt-compiler:$version"
```

Seems to fix the view models injections crashes. IWOMM.

Reply:
Copy pasta strikes again